Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -99,6 +100,27 @@ - (void)dealloc
   [_sockets[socketID] send:message];
 }
 
+RCT_EXPORT_METHOD(sendBlob:(NSDictionary *)blob socketID:(nonnull NSNumber *)socketID)
+{
+  RCTBlobManager *blobManager = [[self bridge] moduleForClass:[RCTBlobManager class]];
```

Comment:
It means we keep the core of the bridge small, and you only need to include the modules you require. It seem like most of the methods here could live on RCTBlobManager itself, if we exposed a way for it to hook into WebSocketModule, e.g. an API on WebSocketModule: setContentHandler:.. forSocketId:...